lib/public/UrlChecker.js

```diff
@@ -66,21 +67,25 @@ UrlChecker.prototype.enqueue = function(url, baseUrl, customData)
 	// Undocumented internal use: enqueue(linkObj)
 	if (typeof url==="object" && !(url instanceof String) && url.broken_link_checker===true)
 	{
-		return this.linkQueue.enqueue(
-		{
-			url: url.url.parsed,
-			data: { customData:customData, linkObj:url }
-		});
+		if (shouldQueryUrl(url.url.parsed.href, this)===true) {
```

Comment:
I put `{}`'s on new lines. Anal, maybe, but it keeps it consistent. I hadn't bothered with a lint, but maybe I should write one to make this easier.